### ISSUE TITLE
Help about the training speed

### ISSUE THREAD

**Issue by YuanShunJie1** [FuChong-cyber/label-inference-attacks#7]
Hello, I am trying to reproduce the results of the defense strategy against label inference attacks on 4090. However, I find that the training speed is too slow. I wonder is there something wrong?

This is the running script:
python vfl_framework.py --use-mal-optim True --use-mal-optim-all False --use-mal-optim-top False -d CIFAR10 --path-dataset /Datasets/CIFAR10 --k 5 --epochs 100 --half 16 --lap-noise True --noise-scale 1e-3

**Comment by FuChong-cyber** [comment]
That's strange. Are you running with pytorch-cpu?
This command works fine on my PC with RTX 3090 x1.

**Comment by YuanShunJie1** [comment]
No, I run it with a GPU 4090.

**Comment by YuanShunJie1** [comment]
Maybe I find the problem. Thanks.
